<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>control_dialog</class>
 <widget class="QDialog" name="control_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>312</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>メイリオ</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Control Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="fill_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>FILL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dump_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>DUMP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="purge_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>PURGE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ignition_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>IGNITION</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
